projects/sonar/src/app/record/document/detail/detail.component.ts

```diff
@@ -182,10 +182,12 @@ export class DetailComponent implements OnDestroy, OnInit {
     this._httpClient.post(`${this._apiService.getEndpointByType('stats', true)}`, data)
     .subscribe(results => {
       const statistics = {};
+      if (results['file-download'] != null) {
       results['file-download'].buckets.map(res => {
           statistics[res.key] = res.unique_count;
         }
       );
```

Comment:
COSMETIC : 
* Great if you adapt the indentation
* single line instruction --> remove bracket : `results['file-download'].buckets.map(res => statistics[res.key] = res.unique_count);`